nieuwsbrief/2019-05-12-moederdag.markdown

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Moederdag"
+date: 2019-02-16 12:00:00
+description: onze tip voor moederdag.
+cover: /news/img/demerstee.jpg
+permalink: /news/2019-02-23-bus-ottignies/
+---
+
```

Comment:
Op zondag 12 mei er niet alleen de Limburgse bekerfinale bij Basket Lummen, maar ook moederdag wordt gevierd! Wil je je moeder op een aangename manier verrassen, dan hebben we deze tip voor je: bij Elektro Schroeyen vind je altijd een bijzonder cadeautje voor haar. Met 850 m² winkeloppervlakte bieden ze een breed gamma in Beeld-Geluid, Huishoudtoestellen, Klein-Elektro en Lifestyle. Wat je ook kiest, je zal je moeder er blij mee kunnen maken. Vermeld bij je aankooop dat je bij Basket Lummen aangesloten bent, zo steun je bovendien onze club.
Meer informatie vind je via deze link: www.schroeyen.be